undertow/src/test/java/org/wildfly/extension/undertow/UndertowTransformersTestCase.java

```diff
@@ -298,8 +298,8 @@ private void addExtraMavenUrls(ModelTestControllerVersion controllerVersion, Leg
             init.addMavenResourceURL(controllerVersion.getMavenGroupId() + ":wildfly-clustering-common:" + controllerVersion.getMavenGavVersion());
             init.addMavenResourceURL(controllerVersion.getMavenGroupId() + ":wildfly-web-common:" + controllerVersion.getMavenGavVersion());
             // The version here appears to be required to be set to the current version of undertow
-            init.addMavenResourceURL("io.undertow:undertow-servlet:2.0.20.Final");
-            init.addMavenResourceURL("io.undertow:undertow-core:2.0.20.Final");
+            init.addMavenResourceURL("io.undertow:undertow-servlet:2.0.22.Final");
+            init.addMavenResourceURL("io.undertow:undertow-core:2.0.22.Final");
```

Comment:
Using the current version of Undertow in the legacy controller isn't good. #12367 removes the need to do it.